skyportal/tests/api/test_candidates.py

```diff
@@ -632,3 +643,62 @@ def test_candidate_list_redshift_range(
     assert status == 200
     assert len(data["data"]["candidates"]) == 1
     assert data["data"]["candidates"][0]["id"] == obj_id1
+
+
+def test_exclude_by_outdated_annotations(
+    annotation_token, view_only_token, public_group, public_candidate, public_candidate2
+):
+    status, data = api(
+        "GET",
+        "candidates",
+        params={"redshiftRange": "(0,0.5)", "groupIDs": f"{public_group.id}"},
```

Comment:
```suggestion
        params={"groupIDs": f"{public_group.id}"},
```

Might want to remove the redshiftRange filter from these queries I think. All the test objects by default have redshift 0.0 so I don't think it actually filters anything, and just in case sometime in the future someone decides to make the ObjFactory randomize the redshifts or something like that and the `public_candidate` is filtered out by the redshift instead of the annotation origin check, you know? I think it's good to touch only the most relevant parts of the code when writing tests to minimize random failures like that.